ci-automation/base_sysexts.sh

```diff
@@ -0,0 +1,16 @@
+# Definitions of base sysexts to be built, for each arch. Used by
+# image.sh and image_changes.sh.
```

Comment:
Why would they differ by arch?

Reply:
Ideally they should not differ and for now they do not. Maybe I could add common, amd64-only and arm64-only lists, just like I do for OEM sysexts. This would avoid the duplication of sysext names, but still be ready for an arch-specific base-sysext if such ever appears. Something I could do in follow-up PR. Or maybe just keep it simple.

Reply:
I think we don't really want to have the images differ, any future amd64-only sysext images could also be external, i.e., pulled in at provisioning time with the logic we have for OEM sysext downloads.

Since it's a simple change to make it arch-specific I think it's better to keep one variable here without having to duplicate something for a use case we don't have.

But just a minor detail, feel free to ignore.

Reply:
I simplified it.